test/crawlers/cheerio_crawler.test.js

```diff
@@ -554,7 +572,9 @@ describe('CheerioCrawler', () => {
 
     describe('proxy', () => {
         let requestList;
+        let responseMock;
         beforeEach(async () => {
+            responseMock = await utilsRequest.requestAsBrowser({ url: 'http://example.com/' });
```

Comment:
Why those changes to unrelated tests?

Reply:
Because we changed the return value of the `_requestFunction`. It does not return the `{ dom, isXml, body ... }` object anymore but it returns the `response stream`. I had to make some mock changes because of that. Just make it simpler and created one real response stream as a mock for all related tests.

Reply:
Yeah, but the point of mocks is to replace possibly inconsistent behavior, such as replacing network requests with pre-configured data. You added a network request that was not there before and named it a mock.

The point of the replaced `_requestFunction` in those affected tests is to **not** make the network request, that would otherwise be done in the crawler.

Please remove the network request again and mock the stream locally.